src/realm-settings/NewClientProfileForm.tsx

```diff
@@ -78,6 +147,39 @@ export const NewClientProfileForm = () => {
               {t("common:cancel")}
             </Button>
           </ActionGroup>
+          {showAddExecutorsForm && (
+            <>
+              <FormGroup
+                label={t("executors")}
+                fieldId="kc-executors"
+                labelIcon={
+                  <HelpItem
+                    helpText={t("realm-settings:executorsHelpText")}
+                    forLabel={t("executorsHelpItem")}
+                    forID={t("executors")}
+                  />
+                }
+              >
+                <Button
+                  id="addExecutor"
+                  component={Link}
+                  variant="link"
+                  className="kc-addExecutor"
+                  // @ts-ignore
+                  to={`/${realm}/realm-settings/clientPolicies`}
```

Comment:
I found out recently that there is [another way](https://www.patternfly.org/v4/components/button/#router-link) to use router links here that is actually type-safe.
```suggestion
                  component={(props) => (
                    <Link
                      {...props}
                      to={`/${realm}/realm-settings/clientPolicies`}
                    ></Link>
                  )}
                  variant="link"
                  className="kc-addExecutor"
```

Reply:
@jonkoops right that is cool, maybe we could make an issue to change this across the codebase?